extensions/cache/src/main/resources/benchmark-analysis.xml

```diff
@@ -56,12 +56,12 @@
       <l:basic-operations-test test-name="on-max-throughput"
                              duration="${test.long.duration:1m}" total-threads="#{ 10 * ${top.iteration} + 10 }">
          <statistics>
-            <default>
+            <basic>
```

Comment:
I don't feel the rename was worth it to break the existing rg3 benchmarks, but I guess it's not a big deal.